src/com/webonise/carbuilder/ColorFactory.java

```diff
@@ -0,0 +1,40 @@
+package com.webonise.carbuilder;
+
+public class ColorFactory extends AbstractCarFactory{
```

Comment:
here i am trying to get the user specified colour object with this interface which will decide the the colour for that car.